build/xtask/Cargo.toml

```diff
@@ -19,7 +19,7 @@ toml = "0.5.6"
 ron = "0.7"
```

Comment:
General comment on the commit message: I think we would want to do the same thing for rv32 as well. Probably we should land this in `rivos/main` and start using it for both rv32 and rv64. Thoughts? @FawazTirmizi @beezow

Reply:
I agree, especially if we want to change the build flow to go from ELF -> {bin, ihex, srec}.

Reply:
ACK on rv32.

I'm not convinced that we actually want to use ELF as the 'reference' format, however, given we're adding additional program headers during elf generation.

My preference would be that we treat ELF as a debug target for QEMU, and continue with binary generation in parallel. Like two parallel trees of image derivation, depending on desired use.